packages/solid/README.md

```diff
@@ -0,0 +1,200 @@
+| `autoResizeCanvas`      | boolean                | true                  | Determines if the canvas should resize automatically to its container                                                                                                                                                                                 |
+| `marker`                | string                 | undefined             | The Lottie named marker to play.                                                                                                                                                                                                                      |
+
+#### RenderConfig
+
+The `renderConfig` object accepts the following properties:
+
+| Property name      | Type   | Default                       | Description             |
+| ------------------ | ------ | ----------------------------- | ----------------------- |
+| `devicePixelRatio` | number | window\.devicePixelRatio \| 1 | The device pixel ratio. |
+
+## Custom Playback Controls
+
+`DotLottieSolid` component makes it easy to build custom playback controls for the animation. It exposes a `dotLottieRefCallback` prop that can be used to get a reference to the [dotLottie](https://github.com/LottieFiles/dotlottie-web/blob/main/packages/web/README.md#apis) web player instance. This instance can be used to control the playback of the animation using the methods exposed by the [dotLottie](https://github.com/LottieFiles/dotlottie-web/blob/main/packages/web/README.md#methods) web player instance.
+
+Here is an example:
+
+```tsx
+import Solid from 'solidjs';
+import { type DotLottie, DotLottieSolid } from 'dotlottie-solid';
+
+const App = () => {
+  const [dotLottie, setDotLottie] = Solid.createSignal<DotLottie | null>(null);
+
+  function play() {
+    dotLottie()?.play();
+  }
+
+  function pause() {
+    dotLottie()?.pause();
+  }
+
+  function stop() {
+    dotLottie()?.stop();
+  }
+
+  return (
+    <DotLottieSolid
+      src="path/to/animation.lottie"
+      loop
+      autoplay
+      dotLottieRefCallback={setDotLottie}
+    />
+    <div>
+      <button onClick={play}>Play</button>
+      <button onClick={pause}>Pause</button>
+      <button onClick={stop}>Stop</button>
+    </div>
+  );
+};
+```
+
+You can find the list of methods that can be used to control the playback of the animation [here](https://github.com/LottieFiles/dotlottie-web/blob/main/packages/web/README.md#methods).
+
+## Listening to Events
+
+`DotLottieSolid` component can receive a `dotLottieRefCallback` prop that can be used to get a reference to the [dotLottie](https://github.com/LottieFiles/dotlottie-web/blob/main/packages/web/README.md#apis) web player instance. This reference can be used to listen to player events emitted by the [dotLottie](https://github.com/LottieFiles/dotlottie-web/blob/main/packages/web/README.md#events) web instance.
+
+Here is an example:
+
+```tsx
+import Solid from 'solidjs';
```

Comment:
> import Solid from 'solidjs';

I think the better example here is to import {createEffect,onCleanup} from `solid-js`

Reply:
Resolved on https://github.com/LottieFiles/dotlottie-web/pull/234/commits/a8ec1a3ea7d3210762a26d225c95a167fdb4d366